migrator/migrations/m_189_to_m_190_vulnerability_requests_add_name/store/previous/store.go

```diff
@@ -0,0 +1,302 @@
+package postgres
```

Comment:
It looks like these are only used for the tests.  I think we should put them in a test package.  either `test/store/previous|updated` or `store/test/previous|updated`